server/service/src/auth.rs

```diff
@@ -202,6 +208,22 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
             PermissionDSL::HasPermission(Permission::Report),
         ]),
     );
+
+    // TODO add permissions from central
```

Comment:
I don't think that there are any permissions for json schema / forms

Reply:
Yes I had my own schema table before I realized there is already one... Plan to rework this at a later stage